code/game/mecha/combat/marauder.dm

```diff
@@ -44,6 +44,32 @@
 	ME = new /obj/item/mecha_parts/mecha_equipment/antiproj_armor_booster(src)
 	ME.attach(src)
 
+/obj/mecha/combat/marauder/ares
+	desc = "Heavy-duty, combat exosuit, adapted from rejected early versions of the Marauder to serve as a biohazard containment exosuit. This model, albeit rare, can be found among civilian populations."
+	name = "Ares"
+	icon_state = "ares"
+	initial_icon = "ares"
+	operation_req_access = list(ACCESS_SECURITY)
+	step_in = 5
+	max_integrity = 450
+	armor = list(melee = 50, bullet = 40, laser = 20, energy = 20, bomb = 20, bio = 100, rad = 60, fire = 100, acid = 100)
+	max_temperature = 40000
+	wreckage = /obj/structure/mecha_wreckage/ares
+	internal_damage_threshold = 25
+	force = 45
```

Comment:
`step_in`, `internal_damage_threshold` and `force` are all the same as the parent `/obj/mecha/combat/marauder`, so they can be removed from the Ares.